src/pages/MemberProfilePage.tsx

```diff
@@ -99,15 +97,21 @@ const MemberProfilePage: React.FC = () => {
 
   return (
     <Container maxW="container.lg" py={8}>
-      <Box p={6} bg={color}>
+      <Box p={6}>
         <VStack spacing={4} align="stretch">
           <HStack spacing={4}>
             <Heading as="h1" size="lg" flex="1">
               Profile
             </Heading>
-            <Button colorScheme="teal" onClick={() => setIsEditing(!isEditing)}>
+            <Button
+              colorScheme="brand"
+              onClick={() => setIsEditing(!isEditing)}
```

Comment:
You might want to do this as well with `setIsEditing((prevEditing) => !prevEditing)` as if you click this twice really quickly it might not work.

Reply:
fixed